tools/release.sh

```diff
@@ -1,13 +1,43 @@
-#!/bin/sh -xe
+#!/bin/bash -xe
```

Comment:
To allow the use of bash string editing features to keep our release script as sane as possible.